### PR TITLE
regenerate tests using spreadsheet TSV

### PR DESCRIPTION
**Pull request by sierra-moxon**

### CODE REVIEW

File: test_cases/DrugCentral_creative.json [hunk 1]
```diff
@@ -50,5 +50,7 @@
     "test_case_predicate_name": null,
     "test_case_predicate_id": null,
     "test_case_input_id": null,
-    "test_case_runner_settings": []
```
Comment (maximusunc) [inline test_cases/DrugCentral_creative.json:53]: I had opened https://github.com/NCATSTranslator/Tests/pull/9 which should handle running a subset of benchmarks while we work on schema updates.

File: test_assets/Asset_1.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
         "description": null,
         "tags": [],
         "test_source": "SMURF",
-        "test_reference": "https://github.com/NCATSTranslator/Feedback/issues/506",
```
Comment (maximusunc) [inline test_assets/Asset_1.json:31]: Is the removal of all these test_references expected?

File: test_cases/DrugCentral_creative.json (hunk 1 above)
Comment (sierra-moxon) [reply test_cases/DrugCentral_creative.json:53]:
```
    "test_case_objective": "QuantitativeTest",
    "test_case_source": null,
    "test_case_predicate_name": null,
    "test_case_predicate_id": null,
    "test_case_input_id": null,
    "test_case_runner_settings": [
        "limit_queries"
    ]
}
```

File: test_assets/Asset_1.json (hunk 2 above)
Comment (sierra-moxon) [reply test_assets/Asset_1.json]: nope, it looks like the spreadsheet column headers changed during the relay.  fixed.

File: test_cases/DrugCentral_creative.json (hunk 1 above)
Comment (maximusunc) [reply test_cases/DrugCentral_creative.json:53]: It doesn't really matter, but I don't think we need the runner settings for this benchmark. It's already a subset in the benchmarks repo.

**Review by maximusunc — APPROVED** [review]
Lots of changes here. I glanced through.

**Pull request merged**